src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language.mdx

```diff
@@ -102,65 +102,113 @@ import queriesnrqlEventDefinitionsQueryBuilder from 'images/queries-nrql_screens
+import queriesnrqlconsole from 'images/queriesNrql_screenshot-full_nrql-console.png'
 
-We've included a NRQL Lessons application on our [opensource.newrelic.com](https://opensource.newrelic.com/projects/newrelic/nr1-learn-nrql) site to help you quickly see NRQL's value and power by using your own data. To learn how to install and use the NRQL Lessons app, watch this short YouTube video (approx. 2:45 minutes).
+import nr1NrqlAlertConditions from 'images/nr1_nrql_alert_conditions.png'
+
+import NerdGraphscreenshot from 'images/apis-and-data_screenshot-crop_NerdGraph-internal-docs.png'
+
+The New Relic Query Language (NRQL) is one of the most versatile and powerful tools you can use to [query your data](/docs/using-new-relic/data/understand-data/query-new-relic-data), but it can seem overwhelming at first glance. Don't worry! Here is some information to give you a good foundational understanding of NRQL, including what it is, how to use it, and some tips and tricks that will help you get the most out of your queries. 
+
+Here's a quick video to help introduce you to using NRQL by showing you how to find a query from a [dashboard](docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/) and modify it in the [query builder](docs/query-your-data/explore-query-data/query-builder/introduction-query-builder). For more detailed information on querying, including a listing of clauses and functions and example queries, see [NRQL syntax, clauses, and functions](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions).
 
 <Video
-  id="3UHEHbprs6I"
+  id="ad6hZPHAsME"
   type="youtube"
 />
 
-Ready to get started? If you haven't already, be sure to [sign up for a New Relic account](https://newrelic.com/signup). It's free, forever.
+If you haven't already, be sure to [sign up for a New Relic account](https://newrelic.com/signup) (it's free, forever). Let's get started!
 
 ## What is NRQL?
 
-NRQL is an acronym of New Relic query language. It's a query language similar to ANSI SQL ([see the syntax](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language#syntax)). You can use NRQL to retrieve detailed New Relic data and get insight into your applications, hosts, and business-important activity. 
-
-Reasons to use NRQL include:
+NRQL is an acronym of New Relic query language. It's a query language similar to ANSI SQL ([see the syntax](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language#syntax)), and is used to retrieve detailed New Relic data and get insight into your applications, hosts, and business-important activity. NRQL can help you:
```

Comment:
Super clear! I wish that these docs had been here when I first started.

Reply:
Thank you!